Cleanup/Name change

jfeiber opened the pull request:
Commented out example code in server.go.

Renamed some stuff.

@joonhyongcho @MJAlperin23

MJAlperin23 commented on db/migrations/20161013155400_buzzerdb.sql:
```diff
@@ -1,23 +1,13 @@
 -- +goose Up
-CREATE TABLE Device (
```
I thought we might want to leave this for now just to test stuff but if you think we are good that works for me

jfeiber replied:
I think we're fine, probably better to test stuff using our actual tables anyway.

MJAlperin23 commented on db/migrations/20161013155400_buzzerdb.sql:
```diff
@@ -59,7 +49,6 @@ CREATE TABLE WebAppUser (
-DROP TABLE Device;
 DROP TABLE Buzzer;
 DROP TABLE HistoricalParties;
 DROP TABLE WebAppUser;
```
also needs to change to new name

MJAlperin23 commented on db/migrations/20161013155400_buzzerdb.sql:
```diff
@@ -59,7 +49,6 @@ CREATE TABLE WebAppUser (
 -- +goose Down
-DROP TABLE Device;
 DROP TABLE Buzzer;
 DROP TABLE HistoricalParties;
```
needs to change to new name

MJAlperin23 commented on models.go:
```diff
@@ -4,25 +4,14 @@ import (
 type ActiveParty struct {
-    activePartyID int
+    id int
     restaurant Restaurant
```
are we sure these foreign keys will work now that you changed restaurantID to just id which they all have

jfeiber replied:
They won't I forgot to commit some more changes to the models file before PRing, it's fixed now

Commit:
Server.go: Commented out example code.
        Models.go: Got rid of devices table, changed some of the names to be more consistent.
        Migrations: terrible practice but I modified the first migration file to reflect the model filechanges.

jfeiber commented:
Replaced by https://github.com/jfeiber/buzzer/pull/6
